src/adapters/fabric/e2eUtils.js

```diff
@@ -733,6 +733,8 @@ async function sendTransaction(context, signedTransaction, invokeStatus, startTi
             errFlag |= TxErrorEnum.OrdererResponseError;
             invokeStatus.SetFlag(errFlag);
             invokeStatus.SetErrMsg(TxErrorIndex.OrdererResponseError,err.toString());
+            invokeStatus.SetStatusFail();
```

Comment:
As stated by the comments above, an error/timeout from the broadcast call doesn't necessarily mean that the ordering failed. It's just an ACK from the orderer, that might get lost. That's why the events will determine the final state of the transaction.